src/assistant/ResultMenu/SuggestionItemTextSecondary.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * Code copied and adapted from cozy-drive
+ *
+ * See source: https://github.com/cozy/cozy-drive/blob/fbe2df67199683b23a40f476ccdacb00ee027459/src/modules/search/components/SuggestionItemTextSecondary.jsx
+ */
+import React from 'react'
+
+import AppLinker from 'cozy-ui/transpiled/react/AppLinker'
+import SuggestionItemTextHighlighted from './SuggestionItemTextHighlighted'
+import useBreakpoints from 'cozy-ui/transpiled/react/providers/Breakpoints'
+import Link from 'cozy-ui/transpiled/react/Link'
+
+const SuggestionItemTextSecondary = ({ text, query, url, slug }) => {
+  const { isMobile } = useBreakpoints()
+
+  if (isMobile || !url) {
+    return <SuggestionItemTextHighlighted text={text} query={query} />
+  }
+
+  const app = { slug }
```

Comment:
Note: For now it is not a problem, but `<AppLink>` may need other props like `app.mobile` to correctly handle some links. This prop is used only [by the flagship app and to open cozy-pass links](https://github.com/cozy/cozy-flagship-app/blob/master/src/libs/functions/openApp.js#L176-L177), so we are safe for now.